bin/commands/updateItem/helpers.js

```diff
@@ -202,23 +214,27 @@ async function setupConfig(qConfig, environmentFilter, reset) {
 
     if (!config.get(`${environment}.accessToken`)) {
```

Comment:
Maybe you could move the content of the if statement and the if statement on line 233 into own function?

Reply:
Great idea. I moved the setting of auth config into its own function: https://github.com/nzzdev/Q-cli/pull/48/commits/8eaad86fd8b722358f1c3a5a013da778e343f8a3